x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/licensing_callout.tsx

```diff
@@ -16,6 +16,7 @@ export enum LICENSING_FEATURE {
   NATIVE_CONNECTOR = 'nativeConnector',
   CRAWLER = 'crawler',
   INFERENCE = 'inference',
+  SEARCH_APPLICATIONS = 'searchApplications',
```

Comment:
Added a new licensing feature. This file might need to be moved to shared, as now it is being used from more locations than new_index. Happy to hear your thoughts!

Reply:
Feel free to move it to shared!